src/main/groovy/edu/oregonstate/mist/api/Error.groovy

```diff
@@ -96,3 +108,10 @@ class Error {
         }
     }
 }
+
+/**
+ * Wrapper object for returning error(s) in a response
+ */
+ class ErrorResultObject {
```

Comment:
Errors must be returned in a top-level array titled "errors" (http://jsonapi.org/format/#errors). I created this object for the following reasons:
-  Not every ResultObjectResponse has a null errors object
- The existing Error objects can be changed in type to this object, yet maintain the same calls. Hopefully, no changes to existing APIs will be necessary if no calls have to be modified.